src/roles/common/tasks/vsd-transfer-truststore.yml

```diff
@@ -8,20 +8,26 @@
   run_once: true
 
 - name: Copy truststore and certificates to stats VSD
-  shell:
-    cmd: /bin/sshpass -p{{ hostvars[stats_vsd].vsd_custom_password | default(vsd_custom_password | default(vsd_default_password)) }} scp -o StrictHostKeyChecking=no truststore.jks {{ stats_vsd }}.jks {{ stats_vsd }}.pem root@{{ stats_vsd }}:/opt/vsd/ejbca/p12/
+  command:
+    cmd: /bin/sshpass -p{{ hostvars[stats_vsd].vsd_custom_password | default(vsd_custom_password | default(vsd_default_password)) }} \
+         scp -o StrictHostKeyChecking=no truststore.jks {{ stats_vsd }}.jks {{ stats_vsd }}.pem root@{{ stats_vsd }}:/opt/vsd/ejbca/p12/
     chdir:  /opt/vsd/ejbca/p12/
   delegate_to: "{{ vsd_hostname_list[0] }}"
   remote_user: "{{ hostvars[vsd_hostname_list[0]].vsd_custom_username | default(vsd_custom_username | default(vsd_default_username)) }}"
   become: "{{ 'no' if hostvars[vsd_hostname_list[0]].vsd_custom_username | default(vsd_custom_username | default(vsd_default_username)) == 'root' else 'yes' }}"
   no_log: "{{ lookup('env', 'METROAE_NO_LOG') or 'true' }}"
   vars:
     ansible_become_pass: "{{ hostvars[vsd_hostname_list[0]].vsd_custom_password | default(vsd_custom_password | default(vsd_default_password)) }}"
+  changed_when: "false"
```

Comment:
`changed_when: False`

no quotes.  It makes a difference.  A string of "false" is actually boolean true.